src/core/src/preprocess/preprocess_steps_impl.cpp

```diff
@@ -99,6 +100,30 @@ void PreStepsList::add_scale_impl(const std::vector<float>& values) {
         "scale " + vector_to_string(values));
 }
 
+void PreStepsList::add_clamp(float min_value, float max_value) {
+    std::string name = "clamp(min " + std::to_string(min_value) + ", max " + std::to_string(max_value) + ")";
+    
+    m_actions.emplace_back(
+        [min_value, max_value](const std::vector<Output<Node>>& nodes,
+                               const std::shared_ptr<Model>& function,
+                               PreprocessingContext& ctxt) {
+            OPENVINO_ASSERT(nodes.size() == 1,
+                            "Can't apply clamp to multi-plane input. Suggesting to convert current image to "
+                            "RGB/BGR color format using 'PreProcessSteps::convert_color'");
+
+            const auto& node = nodes[0];
+            auto element_type = node.get_element_type();
+            OPENVINO_ASSERT(element_type.is_real(),
+                            "Clamp preprocessing can be applied to 'float' inputs. Consider using 'convert_element_type' before clamping. Current type is: ",
+                            element_type);
+
+            auto clamp_op = std::make_shared<ov::opset1::Clamp>(node, min_value, max_value);
```

Comment:
```suggestion
            auto clamp_op = std::make_shared<ov::op::v0::Clamp>(node, min_value, max_value);
```
or use `opset8`, included already:
```suggestion
            auto clamp_op = std::make_shared<ov::opset8::Clamp>(node, min_value, max_value);
```